tests/sentry/integrations/slack/test_disable.py

```diff
@@ -87,6 +87,12 @@ def test_email(self):
             )
             in msg.body
         )
+        assert (
+            self.organization.absolute_url(
+                f"/settings/{self.organization.slug}/integrations/{self.integration.provider}/referrer=disabled-integration/"
```

Comment:
The referrer should have a `?` to make it a query string e.g. https://sentry.sentry.io/alerts/rules/details/5081/?alert=86807&referrer=alert_email